src/client/components/pages/revision.js

```diff
@@ -166,24 +193,47 @@ class RevisionPage extends React.Component {
 			new Date(revision.createdAt).toDateString();
 
 		return (
-			<div>
-				<h1>Revision #{revision.id}</h1>
-				{diffDivs}
-				<p className="text-right">
-					Created by&nbsp;
-					<a
-						href={`/editor/${revision.author.id}`}
-						title={editorTitle}
-					>
-						{revision.author.name}
-					</a>
-					, {dateRevisionCreated}
-				</p>
-
-				<h3>Revision Notes</h3>
-				{revisionNotes}
-
-			</div>
+			<Row>
+				<Col md={12}>
+					<h1>Revision #{revision.id}</h1>
+					{diffDivs}
+					<p className="text-right">
+						Created by&nbsp;
+						<a
+							href={`/editor/${revision.author.id}`}
+							title={editorTitle}
+						>
+							{revision.author.name}
+						</a>
+						, {dateRevisionCreated}
+					</p>
+
+					<h3>Revision Notes</h3>
+					{revisionNotes}
+					{user &&
+						<form
+							className="margin-top-2"
+							onSubmit={this.handleSubmit}
+						>
+							<Input
+								label="Additional Revision Notes"
```

Comment:
I'd call this something like "Add Note", which corresponds to the user interacting with the page, rather than just a display of further information.